docs/timezone.md

```diff
@@ -133,12 +135,77 @@ tz2 = Temporal.TimeZone.from(tz);
 ### timeZone.**id** : string
 
 The `id` property gives an unambiguous identifier for the time zone.
-Effectively, this is the canonicalized version of whatever `timeZoneIdentifier` was passed as a parameter to the constructor.
+This is the normalized version of whatever `timeZoneIdentifier` was passed as a parameter to the constructor.
 
 When subclassing `Temporal.TimeZone`, this property must be overridden to provide an identifier for the custom time zone.
 
 ## Methods
 
+### timeZone.**equals**(_other_: Temporal.TimeZone | object | string) : boolean
+
+**Parameters:**
+
+- `other` (`Temporal.TimeZone` object, object implementing the `Temporal.TimeZone` protocol, or a string time zone identifier): Another time zone to compare.
+
+**Returns:** `true` if `timeZone` and `other` are equal, or `false` if not.
+
+Compares two time zones for equality.
+Equality is determined by the following algorithm:
+
+- If `timeZone === other`, then the time zones are equal.
+- Otherwise, `timeZone.id` is compared to `other` (or `other.id` if `other` is an object).
+  If any of the following conditions are true, then the time zones are equal:
+  - If both string identifiers are Zone or Link names in the [IANA Time Zone Database](https://www.iana.org/time-zones), and they resolve to the same Zone name.
+    This resolution is case-insensitive.
+  - If both string identifiers are custom time zone identifiers and both identifiers are equal.
+    This is a case-sensitive comparison.
+  - If both identifiers are numeric offset time zone identifiers like `"+05:30"`, and they represent the same offset.
+- Otherwise, the time zones are not equal.
+
+Time zones that resolve to different Zones in the IANA Time Zone Database are not equal, even if those Zones use the same offsets.
+Similarly, a numeric-offset identifier is never equal to a named time zone in the IANA Time Zone Database, even if they represent the same offsets.
+
+Although there may be slight variation between implementations, ECMAScript implementations generally build the IANA Time Zone Database using build options that guarantee at least one Zone for every <a href="https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2">ISO 3166-1 Alpha-2</a> country code.
+This behavior differs from the default build options of the IANA Time Zone Database where a Zone may span multiple countries that have shared the same offsets since 1970, for example Europe/Oslo, Europe/Stockholm, Europe/Copenhagen, and Europe/Berlin.
+To avoid conflating different countries' time zones that may vary in the future, ECMAScript does not use these default build options.
```

Comment:
```suggestion
To avoid conflating different countries' time zones that may vary in the future, ECMAScript does not recommend these default build options.
```